serde-reflection/src/format.rs

```diff
@@ -332,11 +341,11 @@ impl<T> Variable<T> {
         Self(Rc::new(RefCell::new(content)))
     }
 
-    fn borrow(&self) -> Ref<Option<T>> {
+    pub fn borrow(&self) -> Ref<Option<T>> {
         self.0.as_ref().borrow()
     }
 
-    fn borrow_mut(&mut self) -> RefMut<Option<T>> {
+    pub fn borrow_mut(&self) -> RefMut<Option<T>> {
```

Comment:
Why the change on mutability? and i'm expecting that you want these functions to be part of the public API?

Reply:
So TBH it's public mostly for unit tests.
On second thought, I made the type more precise (although possibly less intuitive) because I found no reason no too.